mkosi/config.py

```diff
@@ -1964,6 +1964,11 @@ def load_config(args: argparse.Namespace) -> MkosiConfig:
             output = prefix
         args.output = Path(output)
 
+    if "_" in str(args.output) and args.output_format in [OutputFormat.subvolume]:
```

Comment:
Since it's only one and will most likely stay like that.
```suggestion
    if "_" in str(args.output) and args.output_format == OutputFormat.subvolume:
```

Reply:
Thanks for the review. The use of a list here is that, originally, I developed this fix after/on top of another PR of mine  (#1535)- fixing subvolume creation which also reinstates read-only subvolume support by adding an additional `OutputFormat.subvolume_ro`   and initially posted this PR with that included in this list. When I realised I removed the `subvolume_ro` but left the list structure in the expectation that `subvolume_ro` may well exist soon! I can change this though, and move this list construct to the PR that adds `subvolume_ro`